lib/src/shared/main/java/com/couchbase/lite/internal/fleece/MValue.java

```diff
@@ -127,13 +130,14 @@ public void encodeTo(@NonNull FLEncoder enc) {
     @SuppressWarnings("NoFinalizer")
     @Override
     protected void finalize() throws Throwable {
-        nativeChangeSlot(null);
+        nativeChangeSlot(null); // ??? WAT?
```

Comment:
Umm ... I don't remember why? It seems like it will do nothing all the time.

Reply:
BTW, LiteCore or Fleece has provided C API for Mutable Fleece. We should wrap the C Mutable Fleece API instead of implementing everything here.

Reply:
Right!  Will file a ticket to do that!

https://issues.couchbase.com/browse/CBL-694